keeper/keeper.go

```diff
@@ -323,7 +325,10 @@ func (k *Keeper) Deliver(ctx context.Context, bz []byte) error {
 
 // HandlePayload is a utility that handles custom payloads when delivering portal messages.
 func (k *Keeper) HandlePayload(ctx context.Context, payload []byte) error {
-	chain, _ := k.wormhole.GetChain(ctx)
+	chain, err := k.wormhole.GetChain(ctx)
+	if err != nil {
+		return sdkerrors.Wrap(err, "unable to get wormhole chain id")
```

Comment:
nit: the chain name is wormchain
```suggestion
		return sdkerrors.Wrap(err, "unable to get wormchain chain id")
```

Reply:
Good feedback, but in this case, `GetChain` returns the Chain ID given to Noble specifically by the Wormhole ecosystem. Check out the Wormhole documentation [here](https://wormhole.com/docs/build/reference/chain-ids)!